tests/unit/spec/base/datalayer-begincheckout.js

```diff
@@ -14,18 +14,20 @@ import TrackBeginCheckout from '../../../../media/js/base/datalayer-begincheckou
 describe('datalayer-begincheckout.es6.js', function () {
     const expectedObj = {
         event: 'begin_checkout',
-        currency: 'USD',
-        value: '11.88',
-        items: [
-            {
-                item_id: 'testid',
-                item_name: 'email',
-                item_category: 'relay',
-                item_variant: 'yearly',
-                price: '11.88',
-                discount: '12.00'
-            }
-        ]
+        ecommerce: {
```

Comment:
Turned up a bug in this test while adding these ones. Also submitted as https://github.com/mozilla/bedrock/pull/13237